general_rules/Procedure.tex

```diff
@@ -81,12 +81,12 @@ \subsection{Removal of robots}
 \subsection{Start signal}
 \label{rule:start_signal}
 
-Different challenges are started in different ways, according to what would make the most sense in the application setting. 
-Before a challenge starts, robots are waiting in a queue, sometimes accompanied by a team member. 
+Different tests are started in different ways, according to what would make the most sense in the application setting. 
+Before a test starts, robots are waiting in a queue, sometimes accompanied by a team member. 
 
 The various start methods are described below:
 \begin{enumerate}
-	\item \textbf{Door opening:} The robot is waiting behind the door, outside the arena (accompanied by a team member). The challenge starts when the door is opened by a referee, not by a team member. 
+	\item \textbf{Door opening:} The robot is waiting behind the door, outside the arena (accompanied by a team member). The test starts when the door is opened by a referee, not by a team member. 
```

Comment:
I don't get it. Some tests have 3 runs, some others only one, of course teams will try. I see no need to over specify.